apps/root/src/constants/routes.tsx

```diff
@@ -29,3 +29,6 @@ export const TRANSFER_ROUTE = {
   key: 'transfer',
   icon: <TransferIcon />,
 };
+
+export const TOKEN_PROFILE_ROUTE = 'token';
```

Comment:
lets make this an object even tho we dont show it on the navigation to keep consistency